### PR TITLE
MIG-1362: Issue in modules/migration-adding-cluster-to-cam.adoc

### PR DESCRIPTION
**Pull request by anarnold97**
### Jira

* [MIG-1362](https://issues.redhat.com/browse/MIG-1362)

### Version(s):

*  OCP 4.12 → branch/enterprise-4.12
*  OCP 4.13 → branch/enterprise-4.13
*  OCP 4.14 → branch/enterprise-4.14
*  OCP 4.15 → branch/enterprise-4.15
*  OCP 4.16 → branch/enterprise-4.16

Issue:

### Link to docs preview:

* [Adding a cluster to the MTC web console 3-4](https://75456--ocpdocs-pr.netlify.app/openshift-enterprise/latest/migrating_from_ocp_3_to_4/migrating-applications-3-4.html#migration-adding-cluster-to-cam_migrating-applications-3-4)

* [Adding a cluster to the MTC web console](https://75456--ocpdocs-pr.netlify.app/openshift-enterprise/latest/migration_toolkit_for_containers/migrating-applications-with-mtc.html#migration-adding-cluster-to-cam_migrating-applications-with-mtc)

### QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/migration-adding-cluster-to-cam.adoc [hunk 1]
```diff
@@ -59,7 +68,30 @@ $ oc create route passthrough --service=image-registry --port=5000 -n openshift-
...
+  "tags": {
+    "kubernetes.io_cluster.sample-ld57c": "owned",
+    "openshift_creationDate": "2019-10-25T23:28:57.988208+00:00"
+  },
+  "type": "Microsoft.Resources/resourceGroups"
+},
+----
++
+This information is also available from the [Azure Portal](https://portal.azure.com) in the **Resource groups** blade.
 * *Require SSL verification*: Optional: Select this option to verify SSL connections to the cluster.
```
Comment (anarnold97) [inline modules/migration-adding-cluster-to-cam.adoc:95]:
```suggestion
* *Require SSL verification*: Optional: Select this option to verify Secure Socket Layer (SSL) connections to the cluster.
```

File: modules/migration-adding-cluster-to-cam.adoc [hunk 2]
```diff
@@ -59,8 +68,31 @@ $ oc create route passthrough --service=image-registry --port=5000 -n openshift-
 ----
 
 * *Azure cluster*: You must select this option if you use Azure snapshots to copy your data.
-* *Azure resource group*: This field is displayed if *Azure cluster* is selected. Specify the Azure resource group.
-* *Require SSL verification*: Optional: Select this option to verify SSL connections to the cluster.
+** *Azure resource group*: This field is displayed if *Azure cluster* is selected. Specify the Azure resource group.
++
+When an OpenShift cluster is created on Microsoft Azure, an Azure Resource Group is created to contain all resources associated with the cluster. You can display all resource groups by issuing `az group list` in the Azure CLI.
++
+ `ResourceGroups` associated with OpenShift clusters are tagged, so where `sample-rg-name` is the value you would extract and supply to the UI:
```
Comment (anarnold97) [inline modules/migration-adding-cluster-to-cam.adoc:75]:
```suggestion
`ResourceGroups` associated with OpenShift clusters are tagged, so where `sample-rg-name` is the value you would extract and supply to the UI:
```

File: modules/migration-adding-cluster-to-cam.adoc [hunk 3]
```diff
@@ -59,8 +68,31 @@ $ oc create route passthrough --service=image-registry --port=5000 -n openshift-
...
+  "tags": {
+    "kubernetes.io_cluster.sample-ld57c": "owned",
+    "openshift_creationDate": "2019-10-25T23:28:57.988208+00:00"
+  },
+  "type": "Microsoft.Resources/resourceGroups"
+},
+----
++
+This information is also available from the [Azure Portal](https://portal.azure.com) in the **Resource groups** blade.
+* *Require SSL verification*: Optional: Select this option to verify Secure Socket Layer (SSL) connections to the cluster.
```
Comment (anarnold97) [inline modules/migration-adding-cluster-to-cam.adoc:95]:
```suggestion
* *Require SSL verification*: Optional: Select this option to verify the Secure Socket Layer (SSL) connection to the cluster.
```

File: modules/migration-adding-cluster-to-cam.adoc [hunk 4]
```diff
@@ -24,16 +25,24 @@ You can add a cluster to the {mtc-full} ({mtc-short}) web console.
 +
 [source,terminal]
 ----
-$ oc sa get-token migration-controller -n openshift-migration
+$ oc create token migration-controller -n openshift-migration
```
Comment (anarnold97) [inline modules/migration-adding-cluster-to-cam.adoc:28]: @stillalearner - please can I ask you to confirm this change is correct? 

Thanks

**Review by stillalearner — APPROVED** [review]
lgtm

File: modules/migration-adding-cluster-to-cam.adoc [hunk 5]
```diff
@@ -59,8 +68,31 @@ $ oc create route passthrough --service=image-registry --port=5000 -n openshift-
...
+  "tags": {
+    "kubernetes.io_cluster.sample-ld57c": "owned",
+    "openshift_creationDate": "2019-10-25T23:28:57.988208+00:00"
+  },
+  "type": "Microsoft.Resources/resourceGroups"
+},
+----
++
+This information is also available from the [Azure Portal](https://portal.azure.com) in the **Resource groups** blade.
+* *Require SSL verification*: Optional: Select this option to verify the Secure Socket Layer (SSL) connection to the cluster.
```
[the remark this answers is not in the document]
Comment (skopacz1) [reply modules/migration-adding-cluster-to-cam.adoc:95]: @aireilly is this an exception that should be added to the vale bot? Seems like a valid case to use just "SSL".

File: modules/migration-adding-cluster-to-cam.adoc [hunk 6]
```diff
@@ -59,8 +68,31 @@ $ oc create route passthrough --service=image-registry --port=5000 -n openshift-
...
+  },
+  "tags": {
+    "kubernetes.io_cluster.sample-ld57c": "owned",
+    "openshift_creationDate": "2019-10-25T23:28:57.988208+00:00"
+  },
+  "type": "Microsoft.Resources/resourceGroups"
+},
+----
++
+This information is also available from the [Azure Portal](https://portal.azure.com) in the **Resource groups** blade.
```
Comment (skopacz1) [inline modules/migration-adding-cluster-to-cam.adoc:94]: The link to the Azure portal doesn't render correctly in the preview, here's what it should be changed to:
```suggestion
This information is also available from the link:https://portal.azure.com[Azure Portal] in the **Resource groups** blade.
```

Source: [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites)

**Review by skopacz1 — COMMENTED** [review]
Left a few comments, otherwise LGTM!

File: modules/migration-adding-cluster-to-cam.adoc [hunk 7]
```diff
@@ -24,16 +25,24 @@ You can add a cluster to the {mtc-full} ({mtc-short}) web console.
...
+eyJhbGciOiJSUzI1NiIsImtpZCI6IiJ9.eyJpc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW50Iiwia3ViZXJuZXRlcy5pby9zZXJ2a
+WNlYWNjb3VudC9uYW1lc3BhY2UiOiJtaWciLCJrdWJlcm5ldGVzLmlvL3NlcnZpY2VhY2NvdW50L3NlY3JldC5uYW1lIjoibWlnLXRva2V
+uLWs4dDJyIiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9zZXJ2aWNlLWFjY291bnQubmFtZSI6Im1pZyIsImt1YmVybmV0ZXMua
+W8vc2VydmljZWFjY291bnQvc2VydmljZS1hY2NvdW50LnVpZCI6ImE1YjFiYWMwLWMxYmYtMTFlOS05Y2NiLTAyOWRmODYwYjMwOCIsInN
+1YiI6InN5c3RlbTpzZXJ2aWNlYWNjb3VudDptaWc6bWlnIn0.xqeeAINK7UXpdRqAtOj70qhBJPeMwmgLomV9iFxr5RoqUgKchZRG2J2rk
+qmPm6vr7K-cm7ibD1IBpdQJCcVDuoHYsFgV4mp9vgOfn9osSDp2TGikwNz4Az95e81xnjVUmzh-NjDsEpw71DH92iHV_xt2sTwtzftS49L
+pPW2LjrV0evtNBP_t_RfskdArt5VSv25eORl7zScqfe1CiMkcVbf2UqACQjo3LbkpfN26HAioO2oH0ECPiRzT0Xyh-KwFutJLS9Xgghyw-
+LD9kPKcE_xbbJ9Y4Rqajh7WdPYuB0Jd9DPVrslmzK-F6cgHHYoZEv0SvLQi-PO0rpDrcjOEQQ
```
Comment (anarnold97) [inline modules/migration-adding-cluster-to-cam.adoc:42]:
```suggestion
eyJhbGciOiJSUzI1NiIsImtpZCI6IiJ9.eyJpc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW50Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9uYW1lc3BhY2UiOiJtaWciLCJrdWJlcm5ldGVzLmlvL3NlcnZpY2VhY2NvdW50L3NlY3JldC5uYW1lIjoibWlnLXRva2VuLWs4dDJyIiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9zZXJ2aWNlLWFjY291bnQubmFtZSI6Im1pZyIsImt1YmVybmV0ZXMuaW8vc2VydmljZWFjY291bnQvc2VydmljZS1hY2NvdW50LnVpZCI6ImE1YjFiYWMwLWMxYmYtMTFlOS05Y2NiLTAyOWRmODYwYjMwOCIsInN1YiI6InN5c3RlbTpzZXJ2aWNlYWNjb3VudDptaWc6bWlnIn0.xqeeAINK7UXpdRqAtOj70qhBJPeMwmgLomV9iFxr5RoqUgKchZRG2J2rkqmPm6vr7K-cm7ibD1IBpdQJCcVDuoHYsFgV4mp9vgOfn9osSDp2TGikwNz4Az95e81xnjVUmzh-NjDsEpw71DH92iHV_xt2sTwtzftS49LpPW2LjrV0evtNBP_t_RfskdArt5VSv25eORl7zScqfe1CiMkcVbf2UqACQjo3LbkpfN26HAioO2oH0ECPiRzT0Xyh-KwFutJLS9Xgghyw-LD9kPKcE_xbbJ9Y4Rqajh7WdPYuB0Jd9DPVrslmzK-F6cgHHYoZEv0SvLQi-PO0rpDrcjOEQQ
```

File: modules/migration-adding-cluster-to-cam.adoc [hunk 8]
```diff
@@ -59,8 +61,31 @@ $ oc create route passthrough --service=image-registry --port=5000 -n openshift-
 ----
 
 * *Azure cluster*: You must select this option if you use Azure snapshots to copy your data.
-* *Azure resource group*: This field is displayed if *Azure cluster* is selected. Specify the Azure resource group.
-* *Require SSL verification*: Optional: Select this option to verify SSL connections to the cluster.
+** *Azure resource group*: This field is displayed if *Azure cluster* is selected. Specify the Azure resource group.
++
+When an OpenShift cluster is created on Microsoft Azure, an Azure Resource Group is created to contain all resources associated with the cluster. You can display all resource groups by issuing `az group list` in the Azure CLI.
```
Comment (anarnold97) [inline modules/migration-adding-cluster-to-cam.adoc:66]:
```suggestion
When an {OCP} cluster is created on Microsoft Azure, an Azure Resource Group is created to contain all resources associated with the cluster. You can display all resource groups by issuing `az group list` in the Azure CLI.
```

File: modules/migration-adding-cluster-to-cam.adoc [hunk 9]
```diff
@@ -59,8 +61,36 @@ $ oc create route passthrough --service=image-registry --port=5000 -n openshift-
...
 * *Azure cluster*: You must select this option if you use Azure snapshots to copy your data.
-* *Azure resource group*: This field is displayed if *Azure cluster* is selected. Specify the Azure resource group.
-* *Require SSL verification*: Optional: Select this option to verify SSL connections to the cluster.
+** *Azure resource group*: This field is displayed if *Azure cluster* is selected. Specify the Azure resource group.
++
+When an {OCP} cluster is created on Microsoft Azure, an Azure Resource Group is created to contain all resources associated with the cluster. In the Azure CLI, you can display all resource groups by issuing:
++
+[source,terminal]
+----
+az group list
```
Comment (skopacz1) [inline modules/migration-adding-cluster-to-cam.adoc:70]: Just a few changes to make this command formatting follow our docs guidelines:
```suggestion
When an {OCP} cluster is created on Microsoft Azure, an Azure Resource Group is created to contain all resources associated with the cluster. In the Azure CLI, you can display all resource groups by issuing the following command:
+
[source,terminal]
----
$ az group list
```

**Review by skopacz1 — COMMENTED** [review]
One small nit about some command formatting, otherwise once that feedback is addressed this PR is ready to merge. I'll keep an eye out for changes to this PR

File: modules/migration-adding-cluster-to-cam.adoc (hunk 9 above)
Comment (anarnold97) [reply modules/migration-adding-cluster-to-cam.adoc:70]: You are right
image

**Comment by anarnold97** [comment]
> @anarnold97 Can you also squash the commit you made to address my feedback? Thank you

Done, thanks

**Comment by skopacz1** [comment]
Everything is merged except for the 4.14 changes, which are blocked by some unrelated issue in the 4.14 branch. I'm investigating a fix now

**Pull request merged**